content/zh/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -619,7 +619,7 @@ To disallow access, the service would return:
   "kind": "ImageReview",
   "status": {
     "allowed": false,
-    "reason": "image currently blacklisted"
+    "reason": "image currently blocklisted"
```

Comment:
Watch out for changing documentation without also changing the code that generates data. Revising the docs here doesn't fix the code.

Reply:
this one should be safe too.